src/test/java/com/svydovets/bibirnate/entities/EntityWrongType.java

```diff
@@ -0,0 +1,13 @@
+package com.svydovets.bibirnate.entities;
+
+import com.svydovets.bibirnate.annotation.Entity;
+
+import lombok.Data;
+
+@Entity
+@Data
+public class EntityWrongType {
```

Comment:
Are this and entities above for test purposes? If so, I believe we should move them under the test module.

Reply:
Oh, ignore it, missread.